src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-list/nfs-list.component.html

```diff
@@ -29,6 +29,20 @@
                 i18n>Object Gateway</ng-container>
 </ng-template>
 
+<ng-template #protocol
+             let-row="data.row">
+  <span *ngFor="let protocol of row.protocols">
+    <cds-tag size="md">NFSv{{protocol}}</cds-tag>
+  </span>
+</ng-template>
+
+<ng-template #transport
```

Comment:
to be more clear that its a template
```suggestion
<ng-template #transportTpl
```

Reply:
updated